src/serverlist.cpp

```diff
@@ -235,7 +235,23 @@ CServerListManager::CServerListManager ( const quint16  iNPortNum,
             else if ( CurWhiteListAddress.setAddress ( slWhitelistAddresses.at ( iIdx ) ) )
             {
                 vWhiteList << CurWhiteListAddress;
-                qInfo() << qUtf8Printable ( QString ( "Whitelist entry added: %1" ).arg ( CurWhiteListAddress.toString() ) );
+            }
+        }
+    }
+
+    // assume directoryType will get set to AT_CUSTOM
+    if ( !strDirectoryAddress.compare ( "localhost", Qt::CaseInsensitive ) || !strDirectoryAddress.compare ( "127.0.0.1" ) )
```

Comment:
Just a gut feeling and nothing blocking for this PR: It feels like those "are we a directory" checks are repeated in many places and might benefit from being moved to a helper function.

Reply:
There's a method for it -- but it does an extra check on the `directoryType`.  Because we're not _really_ a directory unless `directoryType` is set to `AT_CUSTOM`.  Here, however, we explicitly do not care about `directoryType` - but do care about the address.